sar_pipeline/aws/preparation/burst_utils.py

```diff
@@ -70,10 +94,10 @@ def find_s3_filepaths_from_suffixes(bucket_name, s3_folder, suffixes) -> dict:
     return suffix_to_s3path
 
 
-def get_burst_ids_and_start_times_for_scene_from_asf(
+def get_burst_info_for_scene_from_asf(
     scene: str, burst_prefix: str = "t", lowercase: bool = True
-) -> tuple[list[str], list[datetime]]:
-    """Get the list of burst_ids corresponding to a scene
+) -> dict:
```

Comment:
It might be worth type-hinting the dictionary output. Something like:
`dict[str, dict[str, datetime.datetime|shapely.geom|str]`